### DISCUSSION TITLE
[KR] KT M mobile : Success

### DISCUSSION THREAD

**Discussion by jangboolee** [kyujin-cho/pixel-volte-patch#268] [Carrier-specific Experiences]
This is an expected result, as the documentation clearly states that the patch is only valid for carriers on the U+ network. But I still wanted to test and see if it's possible to get VoLTE working on my Pixel 8 (JP version) with a physical SIM card from KT M Mobile (KT's MVNO).

The patch itself works well, and I was able to enable the setting toggles for both VoLTE and Wi-Fi calling, but the IMS status does not update. This is after calling KT M Mobile's customer service and getting the device OMD registered.

Unfortunate results, but expected results nonetheless.

Regardless, thank you @kyujin-cho for your work on this!

---------

## EDIT 2024-03-03

I gave it another shot after changing my device's OMD to `PTA-VOLTE_EPS`, and it worked! I did have to change the user agent to `LG U+` under the Pixel IMS SIM settings and reboot my device once, but after this, my network stays on LTE even after a voice call. This is fantastic, thank you!

**Comment by benchrevival** [comment]
@jangboolee can a device be OMD registered with KT if bought outside of Korea?

I can't get any signal in my house in Korea and wondering if enabling VoLTE will help. It works fine outside but indoors it struggles to get any reception at all.

**Reply by jangboolee** [reply]
Yes, my Pixel8 is a Japanese version purchased straight from the Google JP online store, and is OMD registered with KT M mobile as `PTA-VOLTE_EPS`.

I had to go through a little bit of trial and error with the OMD type, but setting the OMD as `PTA-VOLTE_EPS`, running the patch, changing the user agent to `LG U+`, then restarting the device enables VoLTE.

A warning that it works great between KT users, but KT <-> SKT does not work, and all I hear from my side is buzzing. No idea how I sound to SKT users, but I can't hear anything from my side. No idea how it works between KT <-> LG U+ though.

**Comment by SrGato001** [comment]
Hey, I have a Pixel 7a (Europe) and I am using KT M mobile, LTE works well. But I have issues with calls, could you help me understand how can I fix it?
What is OMD? Do I just call them and tell them I have a foreign phone, LTE isnt working so I need help registering OMD? And is there a good solution to the issue with the noise between different carriers? if not satisfied with the patch can it be disabled?

**Reply by jangboolee** [reply]
> What is OMD? Do I just call them and tell them I have a foreign phone, LTE isnt working so I need help registering OMD?

Exactly. You need to call `114` and then connect to an agent. Once connected, you need to tell them that you are using a foreign phone and need OMD registration to `PTA-VOLTE_EPS`. Afterwards, if you run the patch, you will have LTE during voice calls.

> And is there a good solution to the issue with the noise between different carriers?

Not one that I know of, unfortunately.

> if not satisfied with the patch can it be disabled?

You can simply toggle the VoLTE option off, although there is also a way to use Shizuku to undo the patch. 

As a side note, although the patch works well and improves the voice call quality drastically when I talk with other KT users, I don't use it anymore and just live with degraded audio quality. I didn't want to have to toggle the VoLTE on and off depending on the carrier of the person I'm calling, and gamble with carriers of people who could call me at any given time.

**Reply by SrGato001** [reply]
Hi, thanks for the quick reply. Do you know if I get a LG u+ sim this patch will work well?
In my case, the small amount of testing I did (with KT M), making calls works good and quick. But receiving calls, take a while to get to my phone, and once I accept the call takes a bit to actually connect the call, do you have a similar experience?

**Reply by jangboolee** [reply]
> Do you know if I get a LG u+ sim this patch will work well?

Yes, if you look at the [compatibility chart](https://github.com/kyujin-cho/pixel-volte-patch/blob/main/docs/compatibility-chart.en.md), it notes LG U+'s VoLTE compatibility as a circle. I've heard that voice calls work well with LG U+ sims, but there's also high idle battery drain. I don't use voice calls that often, so I'd rather just live with the occasional degraded voice call quality and get better battery life.

> But receiving calls, take a while to get to my phone, and once I accept the call takes a bit to actually connect the call

I don't remember this happening back when I had the patch, and it doesn't happen now when I don't have the patch. Maybe it's potentially related to the OMD registration?
